### DISCUSSION TITLE
Can I transfer the label of integrative clustering on scATAC-Seq into other integrative scATAC-Seq？

### DISCUSSION THREAD

**Discussion by WhaleGe** [GreenleafLab/ArchR#1758] [Questions / Documentation]
I now have two integrated scATAC data in hand, one annotated and the other unannotated. I think it is possible to perform tranfer annotation labels in the same way as scRNA? Transfer the annotated scATAC project label to another one.

**Comment by rcorces** [comment]
We dont have any functions that would help with this. The closest thing that we provide is a function to project bulk data into a single-cell dimensionality reduction. So you could export your data as pseudobulks and then re-project it into your other project. Not ideal, I know. The underlying infrastructure all exists within ArchR but a function to explicitly do this hasnt been coded yet.

**Comment by WhaleGe** [comment]
OK, Thank you very much
